myhpi/tests/settings.py

```diff
@@ -1,5 +1,3 @@
-from myhpi.settings import *
-
```

Comment:
This needs to be here

Reply:
Alright, I assume autoflake removed that automatically

Reply:
Yeah, as assumed: The autoflake hook removes this line on commit. I am not sure though, why.
![grafik](https://user-images.githubusercontent.com/44174681/188590925-dd6c66bd-5d90-4bde-82e7-c2d22c54c082.png)

![grafik](https://user-images.githubusercontent.com/44174681/188591068-f3c2f215-2518-432e-9057-b43f232ed6d7.png)

Reply:
Well because it is not used in the file. Interesting that you are the only one that actually uses the recommended pre-commit hooks ^^. Maybe you can add a statement that does nothing in the settings file so the import is not removed?

Reply:
Ohh true xD I did not even question that if it is there, it must be used. I concentrated on the flag that removes/alters start imports, which is not set here.

Well, ofc I use the hook! I am a subservient little angel: If it's in the docs, I try to follow it :b 

I made autoflake ignore this line now